src/main/java/org/springframework/integration/kafka/support/KafkaHeaders.java

```diff
@@ -32,4 +33,8 @@
 
 	public static final String OFFSET = PREFIX + "_offset";
 
+	public static final String NEXT_OFFSET = PREFIX + "nextOffset";
+
+	public static final String ACKNOWLEDGMENT = PREFIX + "acknowledgment";
```

Comment:
I don't see fixes for the extra `_`, so this PR is for the `1.0.x` and we start `1.1.x` afterward.
Correct ?